### PR TITLE
reorganize HCQ and add prophylaxis

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

- Added text about the study on HCQ for post-exposure prophylaxis (#366)
- Rearranged the text with an eye on narrative flow. 
- Added a controversy section to cover all the retractions etc.

### Related issues

#320 

### Suggested reviewers (optional)

A few suggestions tagged in side bar and @rishirajgoel, whom I cannot tag for review!

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [x] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.treatments.md [hunk 1]
```diff
@@ -206,139 +207,160 @@ In one study, CQ inhibited viral entry of SARS-CoV into Vero E6 cells, a cell li
...
+However, the investigators reported an interesting finding revealed in _post hoc_ analysis: controlling for the administration of antivirals yielded an effect of HCQ on the alleviation of symptoms, with a reported hazard ratio of 8.83, although the 95% confidence interval of 1.09 to 71.3 suggests that this analysis may be underpowered and should be interpreted cautiously, as only 28 patients total (14 in each group) met this criterion.
+Additionally, there was a non-significant trend towards a greater number of lymphocytes in the SOC+HCQ group compared to the SOC-alone group (mean of 0.062 versus 0.008; p=0.57).
+Given the improvement in CRP levels and the possible improvement in lymphocyte count, the authors hypothesized that the addition of HCQ to the current SOC could decrease the inflammatory response, which could help to prevent multiorgan failure and death.
+However, one of the key results of this trial was that the 30% of the patients receiving SOC+HCQ reported adverse outcomes compared to 8.8% of patients receiving only SOC.
 The most common adverse outcome in the SOC+HCQ group was diarrhea (10% vs. 0% in the SOC group, p=0.004).
-
-The study had a few limitations.
```
Comment (rando2) [inline content/20.treatments.md:276]: @alavendelm I think I also suggested reworking this text, similar to your feedback in #367

**Comment by soumitagh** [comment]
Great job!

File: content/20.treatments.md [hunk 2]
```diff
@@ -206,139 +207,161 @@ In one study, CQ inhibited viral entry of SARS-CoV into Vero E6 cells, a cell li
...
-Indeed, a larger sample needs to be tested to validate whether lymphocyte counts do increase in patients treated with SOC+HCQ; although the result in the present analysis appeared promising, it lacked statistical significance.
-Additionally, in this study, SOC included the use of antivirals (Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir, and Interferon alfa), which appeared to introduce confounding effects.
-Thus, to better assess the effects of HCQ, an alternate study design needs to be outlined where SOC does not involve the use of anti-virals.
-In this trial, the samples used to test for presence of the SARS-CoV-2 virus were collected from the upper respiratory tract, and the authors indicated that the use of upper respiratory samples may have introduced false negatives (e.g., [@doi:10.1001/jama.2020.3786]); thus, the identification of biomarkers that can be collected non-invasively would be valuable to studies such as this one.
+Their safety analysis indicated that the adverse effects of administering HCQ to mild and moderate COVID-19 cases were manageable.
```
Comment (soumitagh) [inline content/20.treatments.md:300]: The adverse effects of administration of HCQ to mild and moderate COVID-19 cases seemed manageable.

Comment (rando2) [inline content/20.treatments.md:300]:
```suggestion
The adverse effects of administering HCQ to mild and moderate COVID-19 cases were found to be manageable.
```

**Comment by rando2** [comment]
@cgreene or @agitter, because I authored this, if you wouldn't mind being the one to merge, I would appreciate it!

**Pull request merged**